pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -221,6 +237,10 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 		{
 			ObjectMeta: kapi.ObjectMeta{
 				Name: AdminRoleName,
+				Annotations: map[string]string{
+					RoleDescription: "A project manager. If used in a local binding, an admin user will have rights to view any resource in the project and modify any resource in the project except for role creation and quota. If the cluster-admin wants to allow an admin to modify roles, the cluster-admin must create a project-scoped Policy object using JSON.",
+					RoleSomething:   RoleSomethingVal,
```

Comment:
This seems backwards.  Since most people will be adding roles that they want users to use, why wouldn't we have an explicit opt-in for exclusion.